packages/global-styles-ui/src/screen-revisions/use-global-styles-revisions.tsx

```diff
@@ -41,7 +41,7 @@ const SITE_EDITOR_AUTHORS_QUERY = {
 	capabilities: [ 'edit_theme_options' ],
 };
 const DEFAULT_QUERY = { per_page: 100, page: 1 };
-const EMPTY_ARRAY: RawRevision[] = [];
+const EMPTY_ARRAY: [] = [];
```

Comment:
Probably not the best type, but it works and compiles :)

Reply:
That's OK, unless we want to have multiple `EMPTY_ARRAY`s 😉